README.md

```diff
@@ -29,45 +26,21 @@ We've put a demo of Cucumber.js to [run in your browser](http://cucumber.github.
 
 See [CONTRIBUTING.md](CONTRIBUTING.md) for info on contributing to Cucumber.js.
 
-## Prerequisites
-
-* [Node.js](https://nodejs.org) or [io.js](https://iojs.org)
-* [NPM](https://www.npmjs.com)
-
-Cucumber.js is tested on:
-
-* Node.js 5, 4, 0.12, 0.10, and io.js (see [CI builds](https://travis-ci.org/cucumber/cucumber-js))
-* Google Chrome
-* Firefox
-* Safari
-* Opera
-
-See an example of `cucumber-js` [in a browser](http://cucumber.github.io/cucumber-js/)
-or run it locally by following these steps:
-
-* clone the repository
-* `$ npm install`
-* `$ node scripts/server.js`
-* visit `http://localhost:9797`
+## Install
 
-## Usage
-
-### Install
+### Node
 
 Cucumber.js is available as an npm module.
 
-Install globally with:
-
 ``` shell
-$ npm install -g cucumber
+$ npm install cucumber
 ```
 
-Install as a development dependency of your application with:
+### Browser
 
-``` shell
-$ npm install --save-dev cucumber
-```
+* Grab the latest browserified code from the [release](/release) folder
```

Comment:
I've coached some teams who needed to run Cucumber.js on TVs and other weird devices that only expose a web browser. That's pretty unusual stuff but can still be useful.